Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM gliderlabs/alpine:3.3
+
+ENTRYPOINT ["/bin/fluentd-kinesis-forwarder-monitor"]
+
+RUN apk-install ca-certificates
+
+COPY fluentd-kinesis-forwarder-monitor /bin/fluentd-kinesis-forwarder-monitor
```

Comment:
Because of layer caching, I thought it was best practice to move commands that change infrequently to the top of your Dockerfile.